src/keystore.cpp

```diff
@@ -120,8 +116,9 @@ bool CBasicKeyStore::AddWatchOnly(const CScript &dest)
     LOCK(cs_KeyStore);
     setWatchOnly.insert(dest);
     CPubKey pubKey;
-    if (ExtractPubKey(dest, pubKey))
+    if (ExtractPubKey(dest, pubKey)) {
```

Comment:
nit: Those two brackets and the changes below in `RemoveWatchOnly` silently adding parts of #11403 into c85c34e557b3fb24222cb0bc06d658721a16c2fa, intentionally?